external_parser/event_processors/joined_event.h

```diff
@@ -300,6 +300,109 @@ struct ccb_joined_event : public typed_joined_event {
   }
 };
 
+// MOVE TO VW
```

Comment:
this is fine here, don't need to move to VW since it is binary parser specific and vw will have no use for this